docs/source/user-guide/tutorials/build-pkgs-skeleton.rst

```diff
@@ -0,0 +1,385 @@
+===========================================
+Building conda packages with conda skeleton
+===========================================
+
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+This tutorial describes how to quickly build a conda package for
+a Python module that is already available on PyPI.
+
+In the first procedure, building a simple package, you build a
+package that can be installed in any conda environment of the
+same Python version as your root environment. The remaining
+optional procedures describe how to build packages for other
+Python versions and other architectures, as well as how to upload
+packages to your Anaconda.org account.
+
+
+Who is this for?
+=================
+
+This tutorial is for Windows, macOS and Linux users who wish to
+build a conda package from a PyPI package. No prior knowledge of
+conda build or conda recipes is required.
+
+
+.. _before-you-start1:
+
+Before you start
+=================
+
+Before you start, check the :doc:`prerequisites <index>`.
+
+
+.. _conda-build-skeleton:
+
+Building a simple package with conda skeleton pypi
+==================================================
+
+The ``conda skeleton`` command picks up the PyPI package metadata
+and prepares the conda build recipe. The final step is to
+build the package itself and install it into your conda environment.
+
+It is easy to build a skeleton recipe for any Python package that
+is hosted on PyPI, the official third-party software repository
+for the Python programming language.
+
+In this section you are going to use conda skeleton to generate a
+conda recipe, which informs conda build about where the source
+files are located and how to build and install the package.
+
+You'll be working with a package named Pyinstrument_ that is
```

Comment:
pyinstrument is not a good example anymore, unfortunately.  It split up into two components and that makes this tutorial a lot more complicated.  I started to replace this, but got pulled sideways before finishing: https://github.com/conda/conda-docs/pull/514